### ISSUE TITLE
Size of geoblocked entries wrong

### ISSUE THREAD

**Issue by pidoubleyou** [mediathekview/mediathekviewweb#195]
Entries with geoblocking "DE" have a file size between 300 and 400 bytes (KIKA and ARD).
Does your server have a German IP?

The filmlist contains the file size of the "Normal" url. Should the new filmlist format include all file sizes?

**Comment by assarbad** [comment]
Same issue here, however we're also having this issue recently with the "Elefanten-App" which also uses content this way. It tells us (in German) "you have to be in Germany" to which our daughter proclaims quite angrily that she _is_ in German 😁

So there's a chance that the Geoblocking is broken.

**Comment by bagbag** [comment]
@pidoubleyou 
Yes, the server is in Germany. I would appreciate having all sizes in the new filmlist.
